cypress/pages/LoginPage.js

```diff
@@ -0,0 +1,38 @@
+import Page from './Page';
+
+class LoginPage extends Page {
+  constructor() {
+    super('/login');
+    this.emailInput = '[data-testid=EmailInput]';
+    this.passwordInput = '[data-testid=PasswordInput]';
+    this.submitButton = '[data-testid=SubmitButton]';
+    this.errorMessage = '[data-testid=ErrorMessage]';
+  }
+
+  fillEmail(value) {
+    cy.get(this.emailInput)
+      .clear()
+      .type(value);
+
+    return this;
+  }
+
+  fillPassword(value) {
+    cy.get(this.passwordInput)
+      .clear()
+      .type(value);
+
+    return this;
```

Comment:
You could put ```fillPassword``` and ```fillEmail``` into one function, and maybe call it ```fillAndSubmit```, as there is probably very little chance we would want to just fill one or the other.